darwin/cli_functions.py

```diff
@@ -561,7 +562,27 @@ def list_files(dataset_slug: str, statuses: str, path: str, only_filenames: bool
             filters["statuses"] = "new,annotate,review,complete"
         if path:
             filters["path"] = path
-        for file in dataset.fetch_remote_files(filters):
+
+        if not sort_by:
+            sort_by = "updated_at:desc"
+
+        if not _has_valid_format(sort_by):
+            _error(
+                f"Invalid sort_by parameter '{sort_by}'. Correct format is 'attribute:direction', i.e., 'updated_at:desc'."
```

Comment:
Potential typo - I don't think a `,` is needed after `i.e.`

Reply:
> Potential typo - I don't think a `,` is needed after `i.e.`
> But YAY and +1+1+1+1 for helpful error messages with an example fix suggestion; I think that can help end users along their journey nicely
> (Not for this PR; but might be worth centralizing error messages; so its easier to review and keep them up to date over time.)

Yes please, I would love a place for centralized error messages. Right now I have about the same error messages showing the same default, possible values for sort in 3 different places. If tomorrow we add a new sorting attribute, I bet I (or whoever is working here) will forget to update all of 3 of them.